src/main/java/io/confluent/kafkarest/DefaultKafkaRestContext.java

```diff
@@ -37,47 +39,113 @@ public DefaultKafkaRestContext(
       ConsumerManager consumerManager,
       SimpleConsumerManager simpleConsumerManager
   ) {
-    this(config, metadataObserver, producerPool, consumerManager, simpleConsumerManager, null);
+    this(config, metadataObserver, producerPool, consumerManager, simpleConsumerManager, null,
```

Comment:
The first two constructors are only used by tests - it would be better if they were removed IMO.